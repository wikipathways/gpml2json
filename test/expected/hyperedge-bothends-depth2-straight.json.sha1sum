3e0e9903cdd1a2ff06fd27068db2ab912862be00 ?-
