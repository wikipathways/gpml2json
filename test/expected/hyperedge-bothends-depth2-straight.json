{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["idaf905386","id28382a1c","idc088e267","id6d7f3bae"],"drawAs":"rect","gpmlElementName":"Pathway","height":222.47894511414458,"kaavioType":"Group","name":"test case","padding":5,"type":["Pathway"],"width":308.97388914338075,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","dataSourceVersion":"20130715","standardName":"test case","displayName":"test case","textContent":"Name: test case","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=test%20case&doSearch=1","id":"hash://sha256/3287c9f04f9b424940616b5f00c3a97bd9960b9b719917958c330714db70a614"},"entityMap":{"idaf905386":{"id":"idaf905386","borderWidth":1,"zIndex":12288,"points":[{"x":90.92744437969687,"y":78.0748490898938},{"x":278.97388914338075,"y":78.3325698579936}],"color":"#000000","drawAs":"StraightLine","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","burrs":["e0874","a67c1"]},"e0874":{"id":"e0874","attachmentDisplay":{"position":[0.09380500038409093,0],"offset":[0,0]},"drawAs":"none","isAttachedTo":"idaf905386","borderWidth":0,"height":0,"width":0,"zIndex":12288,"type":["Anchor","Burr"],"kaavioType":"Burr","gpmlElementName":"Anchor"},"a67c1":{"id":"a67c1","attachmentDisplay":{"position":[0.9220507933786898,0],"offset":[0,0]},"drawAs":"none","isAttachedTo":"idaf905386","borderWidth":0,"height":0,"width":0,"zIndex":12288,"type":["Anchor","Burr"],"kaavioType":"Burr","gpmlElementName":"Anchor"},"id28382a1c":{"id":"id28382a1c","borderWidth":1,"zIndex":12288,"points":[{"x":175.1244935731248,"y":192.47894511414458},{"isAttachedTo":"e0874","attachmentDisplay":{"position":[0.09380500038409093,0],"offset":[],"relativeOffset":[0,0]},"x":108.56714120298116,"y":78.09902458664439,"orientation":[]}],"color":"#000000","drawAs":"StraightLine","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerEnd":"MimCatalysis","isAttachedTo":["idaf905386"],"burrs":["f4b98","fb36b"]},"f4b98":{"id":"f4b98","attachmentDisplay":{"position":[0.09919868499280665,0],"offset":[0,0]},"drawAs":"none","isAttachedTo":"id28382a1c","borderWidth":0,"height":0,"width":0,"zIndex":12288,"type":["Anchor","Burr"],"kaavioType":"Burr","gpmlElementName":"Anchor"},"fb36b":{"id":"fb36b","attachmentDisplay":{"position":[0.5185650145522129,0],"offset":[0,0]},"drawAs":"none","isAttachedTo":"id28382a1c","borderWidth":0,"height":0,"width":0,"zIndex":12288,"type":["Anchor","Burr"],"kaavioType":"Burr","gpmlElementName":"Anchor"},"idc088e267":{"id":"idc088e267","borderWidth":1,"zIndex":12288,"points":[{"isAttachedTo":"a67c1","attachmentDisplay":{"position":[0.9220507933786898,0],"offset":[],"relativeOffset":[0,0]},"x":264.31581796609356,"y":78.31248072859039,"orientation":[]},{"isAttachedTo":"f4b98","attachmentDisplay":{"position":[0.09919868499280665,0],"offset":[],"relativeOffset":[0,0]},"x":168.52209174140367,"y":181.13260740823483,"orientation":[]}],"color":"#000000","drawAs":"StraightLine","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerEnd":"MimInhibition","isAttachedTo":["idaf905386","id28382a1c"],"burrs":["eda1f"]},"eda1f":{"id":"eda1f","attachmentDisplay":{"position":[0.4,0],"offset":[0,0]},"drawAs":"none","isAttachedTo":"idc088e267","borderWidth":0,"height":0,"width":0,"zIndex":12288,"type":["Anchor","Burr"],"kaavioType":"Burr","gpmlElementName":"Anchor"},"id6d7f3bae":{"id":"id6d7f3bae","borderWidth":1,"zIndex":12288,"points":[{"isAttachedTo":"fb36b","attachmentDisplay":{"position":[0.5185650145522129,0],"offset":[],"relativeOffset":[0,0]},"x":140.6101791727445,"y":133.16551996132048,"orientation":[]},{"isAttachedTo":"eda1f","attachmentDisplay":{"position":[0.4,0],"offset":[],"relativeOffset":[0,0]},"x":225.9983274762176,"y":119.44053140044817,"orientation":[]}],"color":"#000000","drawAs":"StraightLine","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerEnd":"MimInhibition","isAttachedTo":["id28382a1c","idc088e267"]}}}
